suite-native/module-accounts-import/src/components/AccountImportLoader.tsx

```diff
@@ -1,41 +1,76 @@
-import React from 'react';
+/* eslint-disable global-require */
+import React, { useEffect } from 'react';
+import { Image } from 'react-native';
+import Animated, {
+    useAnimatedStyle,
+    withDelay,
+    withSequence,
+    withTiming,
+} from 'react-native-reanimated';
 
-import { Box, Text } from '@suite-native/atoms';
+import { Box, Loader, Text, VStack } from '@suite-native/atoms';
 import { prepareNativeStyle, useNativeStyles } from '@trezor/styles';
 
-const fetchingStyle = prepareNativeStyle(() => ({
+type AccountImportLoaderProps = {
+    onLoad(): void;
```

Comment:
this is weird. The type should be `onLoad: () => void`

Reply:
Yes, this is exactly what it does. This is a shorter notation `() => void`.

Reply:
Yes, I understand what it does but it's used differently everywhere else in the code base. I wouldn't change this just two save 2 characters. Stay consistent with how it's used or refactor other usages

Reply:
I think this is not a big deal. Same is we agreed that using both 
```
if(cond){
 statement
}
```
and
```
if(cond) statement
```
is both okay.  Why not allow this? I believe that both notations have the same readability.  Maybe just let's agree to not mix these two inside a single definition of type alias/interface.

Reply:
It's just a nit, so if you prefer it so much that it's worth the discussion, you can keep it. It's just weird to use it in combination with arrow function but I don't want to spend time arguing about this so I'll leave it up to you.